consensus.asciidoc

```diff
@@ -4,8 +4,12 @@
 
 When it comes to decentralized record keeping, which is how blockchain technology is often described, it becomes infeasible to rely on trust alone to ensure that factual information is added to the record books. This problem arises in decentralized networks as a result of the lack of a central entity that would typically decide what should and should not be considered as fact. It's difficult to avoid this issue when working with blockchain technology because it is this lack of a central entity that is one of the main contributors to blockchain's popularity. In order to circumvent this issue a network must have a system in place by which a consensus can be reached by the nodes in the network.
 
+//Philosofical: In effect, you are trusting the system. I never appreciated the idea of a "trustless" system, instead one might look at it as shifting the trust in someone to trust in the system.
```

Comment:
I like so much this rethink about _"trustless system"_. I do not know if my opinion will have to do with the fact that English is not my native language. 

I also agree with the considerations that Andreas makes in some of his talks, about which little effort is made to find **terms better adapted** to innovations and how there are tends to reuse old terms that do not faithfully reflect the underlying concept.